documentation/en/cli-lotus.md

```diff
@@ -1114,7 +1178,23 @@ OPTIONS:
    --help, -h    show help
 ```
 
+### lotus mpool clear
```

Comment:
this is new because it's `Hidden` - because it's both deprecated and pretty unsafe to invoke so we don't want to encourage it